framework/components/AFeatureDiscovery/components/ATourStep.js

```diff
@@ -0,0 +1,71 @@
+import React, {useEffect, useRef, forwardRef} from "react";
+
+import APopover from "../../APopover";
+import ATourTooltip from "./ATourTooltip";
+
+import ADialog from "../../ADialog";
+
+const ATourStep = forwardRef(({
+  children,
+  rootRef,
+  isLastStep,
+  index,
+  step,
+  role,
+  onNext,
+  onPrev,
+  onClose,
+  ...rest
+}, ref) => {
+  const {placement, targetSelector} = step;
+
+  const target = rootRef.current.querySelector(targetSelector);
+  const anchorRef = useRef();
+  anchorRef.current = target;
+
+  useEffect(() => {
+    document.querySelector("html").style.overflow = "hidden";
+    return () => (document.querySelector("html").style.overflow = "unset");
+  }, []);
+
+  const tooltipProps = {
+    index,
+    step,
+    isLastStep,
+    onNext,
+    onPrev,
+    onClose
+  };
+  const tooltipComponent = children ? (
+    children(tooltipProps)
+  ) : (
+    <ATourTooltip {...tooltipProps}></ATourTooltip>
+  );
+  
+  if (placement === "center") {
+    return (
+      <ADialog ref={ref} open={true} {...rest}>
+        {tooltipComponent}
+      </ADialog>
+    );
+  }
+
+  return (
+    <APopover
```

Comment:
If you use `AMenu` instead of `APopover`, the arrows will be fixed: 
![image](https://user-images.githubusercontent.com/14955351/147268212-dce3b032-aa71-4268-8d0d-72974afceca9.png)